alena_automation/XPathExercises - Alena2.csv

```diff
@@ -0,0 +1,34 @@
+Xpath Tasks,Authors.xml,Authors2.xml (with namespaces)
+Normal:,,
+all elements,/authors,
```

Comment:
/authors returns only 1 element - "authors"
need smth to return all elements (authors, all author, all name, all nationality)

Reply:
/*  -- am I right?